Code/src/test/java/nl/utwente/group10/haskell/expr/ApplyTest.java

```diff
@@ -1,64 +1,95 @@
 package nl.utwente.group10.haskell.expr;
 
-import nl.utwente.group10.haskell.exceptions.HaskellException;
+import nl.utwente.group10.haskell.env.Env;
+import nl.utwente.group10.haskell.exceptions.HaskellTypeError;
+import nl.utwente.group10.haskell.hindley.GenSet;
 import nl.utwente.group10.haskell.type.*;
 import org.junit.Before;
 import org.junit.Test;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.*;
 
 public class ApplyTest {
-    private Type T_integer;
-    private Type T_float;
-    private Type T_string;
-    private Type T_numeric;
-    private Type T_a;
-    private Type T_b;
-    private Type T_c;
+    private final Type alpha = new VarT("a");
+    private final Type beta = new VarT("b");
+    private final Type gamma = new VarT("c");
+    private final Type alphaList = new ListT(this.alpha);
+    private final Type betaList = new ListT(this.beta);
+    private final Type integer = new ConstT("Int");
+    private final Type integerList = new ListT(this.integer);
+    private final Type string = new ConstT("Str");
```

Comment:
The Haskell type is really `String`, which is an alias for `[Char]` - a different example might be clearer.